Nicky/HW3/R_OSPool.submit

```diff
@@ -0,0 +1,27 @@
+SingularityImage = "/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-r:3.5.0"
+
+# Specify your executable and arguments
+executable = spill_calculation.r
+arguments = $(spill)
+
+# Tell HTCondor what to name log, standard error, and standard output("screen output") files
+log = log/spill$(Process).log
+output = output/spill$(Process).out
```

Comment:
Hi @nickytagoe, please change the output and error location to *errout* as mentioned in the HW3. Just make changes here..you don't need to run the calculations again.